data/101/748/149/101748149.geojson

```diff
@@ -4,21 +4,24 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
-    "geom:area":0.004168,
-    "geom:area_square_m":27011086.576898,
-    "geom:bbox":"24.4456698711,58.3415022222,24.5830479577,58.4322007565",
-    "geom:latitude":58.389681,
-    "geom:longitude":24.513518,
+    "geom:area":0.005088,
+    "geom:area_square_m":32981233.886581,
+    "geom:bbox":"24.451272,58.341612,24.604438,58.416391",
+    "geom:latitude":58.38389,
+    "geom:longitude":24.525913,
     "gn:country":"EE",
     "gn:fcode":"PPLA",
+    "gn:gn_country":"EE",
```

Comment:
These duplicate existing `gn:*` properties.

Reply:
In the superseding of duplicates, I brought over source (`ne`, `woe`, `qs`, etc properties, as well as `name` properties from the record being superseded.. in this case, this record supersedes `1125999409`, which had the  `"gn:gn_country"` property.

The fact that we store both `gn:country` and `gn:gn_country` properties doesn't help. Worth going through the repo to deduplicate these source properties?

Reply:
Fun times. File followup issue to sort out.

Reply:
Filed https://github.com/whosonfirst-data/whosonfirst-data/issues/1876 to sort this out.